src/Components/Onboarding/FindVideoModal.jsx

```diff
@@ -4,7 +4,7 @@ const FindVideoModal = () => {
     return (
         <div
             // eslint-disable-next-line react/no-unknown-property
-            container
+            container="true"
```

Comment:
Do we need the `true` here? I thought that if an attribute was present it would evaluate to truthy

Reply:
it doesn't, I was getting an error in my browser console!